extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -110,8 +110,8 @@ export async function exportResultsToGist(
 const buildGistDescription = (query: RemoteQuery, analysesResults: AnalysisResults[]) => {
   const resultCount = sumAnalysesResults(analysesResults);
   const resultLabel = pluralize(resultCount, 'result', 'results');
-  const repositoryLabel = pluralize(query.repositoryCount, 'repository', 'repositories');
-  return `${query.queryName} (${query.language}) ${resultLabel} (${repositoryLabel})`;
+  const repositoryLabel = query.repositoryCount ? `(${pluralize(query.repositoryCount, 'repository', 'repositories')})` : '';
```

Comment:
This condition will be satisfied if `repositoryCount` is zero or undefined. I think zero is a weird and unlikely case, so this looks good here. If you want to actually show `(0 repositories)` then explicitly add `=== undefined` in your check.

Reply:
Ah, good point! We shouldn't ever have `0 repositories` (I think), but it's probably worth being explicit 😎